src/components/List/List.tsx

```diff
@@ -151,6 +151,7 @@ export const List = React.forwardRef<HTMLUListElement, Props>(
                 tabIndex="0"
                 onKeyDown={handleKeyDown}
                 {...block({ isDivided, ...rest })}
+                role="listbox"
```

Comment:
:+1:

Reply:
You are right. Role list is for static lists, while listbox is for selectable lists, which is what this component is about. Since we have onKeyDown and tabindex we should use listbox here, but not in the first render block.